components/googleMap/googleMap.scss

```diff
@@ -0,0 +1,26 @@
+.ui-google-map__custom-marker {
```

Comment:
yes, but you can easily avoid using it and this will make file even better (with pure flat structure)